LRDOCS-3717

dnebing opened the pull request:
LRDOCS-3717: Blade samples for creating a Liferay MVC portlet using classic WAR packaging

dnebinger committed:
LRDOCS-3717 Add files for a liferay MVC portlet war

lrbladejenkins commented:
Bundles Diff Test failed on LAX linux server on 4ec89e0abfaeb45147ee94935d1bd4d7f6ae59d5

[link to jenkins](http://cloud-10-50-0-245:8081/job/liferay-blade-samples-pr-tester/448/)

gamerson commented:
Hey Dave,

The build DIFF test doesn't like the fact that the projects are named
differently "jsp-portlet-war" vs "jsp-portlet".  I know that inside of
"liferay-workspace" putting a "-war" extension is redundent, but for now, I
think it is OK.

So can you rename the "jsp-portlet" inside the "liferay workspace" to
"jsp-portlet-war"?

Thanks!


On Wed, Jul 26, 2017 at 9:02 AM, lrbladejenkins <notifications@github.com>
wrote:

> Bundles Diff Test failed on LAX linux server on 4ec89e0
> <https://github.com/gamerson/liferay-blade-samples/commit/4ec89e0abfaeb45147ee94935d1bd4d7f6ae59d5>
>
> link to jenkins
> <http://cloud-10-50-0-245:8081/job/liferay-blade-samples-pr-tester/448/>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/gamerson/liferay-blade-samples/pull/304#issuecomment-318062442>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAkVFZqAoricS0JNj1tVOZcgXBaI13sgks5sR0cSgaJpZM4Oj9s4>
> .
>



-- 
Greg Amerson
Liferay Backend Developer Tools
Liferay, Inc. www.liferay.com

dnebing commented:
Yep, I’ll fix and resend.  I needed “-war” for the apps directory because the “jsp-portlet” was already taken.

 

However, if you have a better name, I can use that instead.

 

 

From: Gregory Amerson <notifications@github.com>
Reply-To: gamerson/liferay-blade-samples <reply@reply.github.com>
Date: Wednesday, July 26, 2017 at 10:19 AM
To: gamerson/liferay-blade-samples <liferay-blade-samples@noreply.github.com>
Cc: Dave Nebinger <61537112@liferay.com>, Author <author@noreply.github.com>
Subject: Re: [gamerson/liferay-blade-samples] LRDOCS-3717 (#304)

 

Hey Dave,

The build DIFF test doesn't like the fact that the projects are named
differently "jsp-portlet-war" vs "jsp-portlet". I know that inside of
"liferay-workspace" putting a "-war" extension is redundent, but for now, I
think it is OK.

So can you rename the "jsp-portlet" inside the "liferay workspace" to
"jsp-portlet-war"?

Thanks!


On Wed, Jul 26, 2017 at 9:02 AM, lrbladejenkins <notifications@github.com>
wrote:

> Bundles Diff Test failed on LAX linux server on 4ec89e0
> <https://github.com/gamerson/liferay-blade-samples/commit/4ec89e0abfaeb45147ee94935d1bd4d7f6ae59d5>
>
> link to jenkins
> <http://cloud-10-50-0-245:8081/job/liferay-blade-samples-pr-tester/448/>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/gamerson/liferay-blade-samples/pull/304#issuecomment-318062442>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAkVFZqAoricS0JNj1tVOZcgXBaI13sgks5sR0cSgaJpZM4Oj9s4>
> .
>



-- 
Greg Amerson
Liferay Backend Developer Tools
Liferay, Inc. www.liferay.com


—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.

gamerson commented:
jsp-portlet-war sounds good to me.  If we think of something better down
the road we can rename it.

On Wed, Jul 26, 2017 at 9:22 AM, Dave Nebinger <notifications@github.com>
wrote:

> Yep, I’ll fix and resend.  I needed “-war” for the apps directory because
> the “jsp-portlet” was already taken.
>
>
>
> However, if you have a better name, I can use that instead.
>
>
>
>
>
> From: Gregory Amerson <notifications@github.com>
> Reply-To: gamerson/liferay-blade-samples <reply@reply.github.com>
> Date: Wednesday, July 26, 2017 at 10:19 AM
> To: gamerson/liferay-blade-samples <liferay-blade-samples@
> noreply.github.com>
> Cc: Dave Nebinger <61537112@liferay.com>, Author <
> author@noreply.github.com>
> Subject: Re: [gamerson/liferay-blade-samples] LRDOCS-3717 (#304)
>
>
>
> Hey Dave,
>
> The build DIFF test doesn't like the fact that the projects are named
> differently "jsp-portlet-war" vs "jsp-portlet". I know that inside of
> "liferay-workspace" putting a "-war" extension is redundent, but for now, I
> think it is OK.
>
> So can you rename the "jsp-portlet" inside the "liferay workspace" to
> "jsp-portlet-war"?
>
> Thanks!
>
>
> On Wed, Jul 26, 2017 at 9:02 AM, lrbladejenkins <notifications@github.com>
> wrote:
>
> > Bundles Diff Test failed on LAX linux server on 4ec89e0
> > <https://github.com/gamerson/liferay-blade-samples/commit/
> 4ec89e0abfaeb45147ee94935d1bd4d7f6ae59d5>
> >
> > link to jenkins
> > <http://cloud-10-50-0-245:8081/job/liferay-blade-samples-pr-tester/448/>
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/gamerson/liferay-blade-samples/pull/
> 304#issuecomment-318062442>,
> > or mute the thread
> > <https://github.com/notifications/unsubscribe-auth/
> AAkVFZqAoricS0JNj1tVOZcgXBaI13sgks5sR0cSgaJpZM4Oj9s4>
> > .
> >
>
>
>
> --
> Greg Amerson
> Liferay Backend Developer Tools
> Liferay, Inc. www.liferay.com
>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/gamerson/liferay-blade-samples/pull/304#issuecomment-318068215>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAkVFd_-Qjt67vb1yDlYIRgSCweAB2UHks5sR0uQgaJpZM4Oj9s4>
> .
>



-- 
Greg Amerson
Liferay Backend Developer Tools
Liferay, Inc. www.liferay.com

dnebinger committed:
Rename files to make jenkins build diff test happy

dnebing commented:
I’ve updated the LRDOCS-3717 PR but cannot resend since the original is still open.  Want to try kicking Jenkins’ tires?

 

From: Gregory Amerson <notifications@github.com>
Reply-To: gamerson/liferay-blade-samples <reply@reply.github.com>
Date: Wednesday, July 26, 2017 at 10:33 AM
To: gamerson/liferay-blade-samples <liferay-blade-samples@noreply.github.com>
Cc: Dave Nebinger <61537112@liferay.com>, Author <author@noreply.github.com>
Subject: Re: [gamerson/liferay-blade-samples] LRDOCS-3717 (#304)

 

jsp-portlet-war sounds good to me. If we think of something better down
the road we can rename it.

On Wed, Jul 26, 2017 at 9:22 AM, Dave Nebinger <notifications@github.com>
wrote:

> Yep, I’ll fix and resend. I needed “-war” for the apps directory because
> the “jsp-portlet” was already taken.
>
>
>
> However, if you have a better name, I can use that instead.
>
>
>
>
>
> From: Gregory Amerson <notifications@github.com>
> Reply-To: gamerson/liferay-blade-samples <reply@reply.github.com>
> Date: Wednesday, July 26, 2017 at 10:19 AM
> To: gamerson/liferay-blade-samples <liferay-blade-samples@
> noreply.github.com>
> Cc: Dave Nebinger <61537112@liferay.com>, Author <
> author@noreply.github.com>
> Subject: Re: [gamerson/liferay-blade-samples] LRDOCS-3717 (#304)
>
>
>
> Hey Dave,
>
> The build DIFF test doesn't like the fact that the projects are named
> differently "jsp-portlet-war" vs "jsp-portlet". I know that inside of
> "liferay-workspace" putting a "-war" extension is redundent, but for now, I
> think it is OK.
>
> So can you rename the "jsp-portlet" inside the "liferay workspace" to
> "jsp-portlet-war"?
>
> Thanks!
>
>
> On Wed, Jul 26, 2017 at 9:02 AM, lrbladejenkins <notifications@github.com>
> wrote:
>
> > Bundles Diff Test failed on LAX linux server on 4ec89e0
> > <https://github.com/gamerson/liferay-blade-samples/commit/
> 4ec89e0abfaeb45147ee94935d1bd4d7f6ae59d5>
> >
> > link to jenkins
> > <http://cloud-10-50-0-245:8081/job/liferay-blade-samples-pr-tester/448/>
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/gamerson/liferay-blade-samples/pull/
> 304#issuecomment-318062442>,
> > or mute the thread
> > <https://github.com/notifications/unsubscribe-auth/
> AAkVFZqAoricS0JNj1tVOZcgXBaI13sgks5sR0cSgaJpZM4Oj9s4>
> > .
> >
>
>
>
> --
> Greg Amerson
> Liferay Backend Developer Tools
> Liferay, Inc. www.liferay.com
>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/gamerson/liferay-blade-samples/pull/304#issuecomment-318068215>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAkVFd_-Qjt67vb1yDlYIRgSCweAB2UHks5sR0uQgaJpZM4Oj9s4>
> .
>



-- 
Greg Amerson
Liferay Backend Developer Tools
Liferay, Inc. www.liferay.com


—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.

lrbladejenkins commented:
Bundles Diff Test failed on LAX linux server on 2818d903c2e79689157dd15ca847fce1e1e75fc5

[link to jenkins](http://cloud-10-50-0-245:8081/job/liferay-blade-samples-pr-tester/449/)

gamerson commented:
hey @lawrence-lee  can you help Dave figure out why the bundles DIFF is failing?

dnebing commented:
I rebased from upstream before creating my branch, so I don’t think there’s anything there.

 

I added 3 new folders, now all named “jsp-portlet-war” to introduce a war-based Liferay MVC portlet, added to the gradle, liferay-workspace and maven areas.

 

Builds are all good of code alone on my end.

 

I don’t have visibility into the Jenkins job to see what failure is occurring.

 

 

From: Gregory Amerson <notifications@github.com>
Reply-To: gamerson/liferay-blade-samples <reply@reply.github.com>
Date: Wednesday, July 26, 2017 at 11:35 AM
To: gamerson/liferay-blade-samples <liferay-blade-samples@noreply.github.com>
Cc: Dave Nebinger <61537112@liferay.com>, Author <author@noreply.github.com>
Subject: Re: [gamerson/liferay-blade-samples] LRDOCS-3717 (#304)

 

hey @lawrence-lee can you help Dave figure out why the bundles DIFF is failing?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.

gamerson committed:
include jsp-portlet-war into gradle/maven builds

gamerson commented:
Oh I see.  In gradle/settings.gradle you have to include your new project.
Same thing in maven/pom.xml    I will make this change and push it to your
remote repo.

G

On Wed, Jul 26, 2017 at 10:39 AM, Dave Nebinger <notifications@github.com>
wrote:

> I rebased from upstream before creating my branch, so I don’t think
> there’s anything there.
>
>
>
> I added 3 new folders, now all named “jsp-portlet-war” to introduce a
> war-based Liferay MVC portlet, added to the gradle, liferay-workspace and
> maven areas.
>
>
>
> Builds are all good of code alone on my end.
>
>
>
> I don’t have visibility into the Jenkins job to see what failure is
> occurring.
>
>
>
>
>
> From: Gregory Amerson <notifications@github.com>
> Reply-To: gamerson/liferay-blade-samples <reply@reply.github.com>
> Date: Wednesday, July 26, 2017 at 11:35 AM
> To: gamerson/liferay-blade-samples <liferay-blade-samples@
> noreply.github.com>
> Cc: Dave Nebinger <61537112@liferay.com>, Author <
> author@noreply.github.com>
> Subject: Re: [gamerson/liferay-blade-samples] LRDOCS-3717 (#304)
>
>
>
> hey @lawrence-lee can you help Dave figure out why the bundles DIFF is
> failing?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/gamerson/liferay-blade-samples/pull/304#issuecomment-318092570>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAkVFYnjDe-bxhk6_HQSnqeUsGgxVpcTks5sR12bgaJpZM4Oj9s4>
> .
>



-- 
Greg Amerson
Liferay Backend Developer Tools
Liferay, Inc. www.liferay.com

lrbladejenkins commented:
Bundles Diff Test failed on LAX linux server on 37398a748d799b0f71cd18159e5e73b71df38ffd

[link to jenkins](http://cloud-10-50-0-245:8081/job/liferay-blade-samples-pr-tester/450/)

gamerson commented:
Pull request submitted to https://github.com/lawrence-lee/liferay-blade-samples/pull/32. [See changes here.](https://github.com/gamerson/liferay-blade-samples/compare/37398a7~1...37398a7) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

gamerson commented:
@dnebing  I'm sending to Lawrence to fix the NPE in the bundle tests.  Once he fixes that, there may be actual "diffs" that you will need to help us look at the diffs.

lawrence-lee commented:
https://github.com/gamerson/liferay-blade-samples/pull/306
